### PR TITLE
ci-images-mirror: fix a few things

### PR DESCRIPTION
**Pull request by hongkailiu**
/assign @deepsm007

### CODE REVIEW

File: pkg/controller/quay_io_ci_images_distributor/quay_io_ci_images_distributor.go [hunk 1]
```diff
@@ -245,7 +245,7 @@ func testInputImageStreamTagFilterFactory(
 	l = logrus.WithField("subcomponent", "test-input-image-stream-tag-filter")
 	return func(nn types.NamespacedName) bool {
 		if ignoreImageStreamTags.Has(nn.String()) {
-			logrus.WithField("tag", nn.String()).Fatal("Ignored events of image stream tag")
```
Comment (hongkailiu) [inline pkg/controller/quay_io_ci_images_distributor/quay_io_ci_images_distributor.go:248]: Oh Boy!
This is a very bad one from a temporary code for testing.
It explains lots of things.

File: cmd/ci-images-mirror/main.go [hunk 2]
```diff
@@ -306,13 +302,21 @@ func main() {
...
 			logrus.WithError(err).Fatal("failed to register metrics")
 		}
+		ignoreImageStreamTags := sets.New[string](opts.quayIOCIImagesDistributorOptions.ignoreImageStreamTagsRaw.Strings()...)
+		if opts.config != nil {
+			for k := range opts.config.SupplementalCIImages {
+				logrus.WithField("target", k).Debug("Ignore target of supplemental CI images on mirroring")
+				ignoreImageStreamTags.Insert(k)
+			}
+		}
+		logrus.WithField("tags", ignoreImageStreamTags.UnsortedList()).Infof("%s will ignore mirroring those tags", quayiociimagesdistributor.ControllerName)
```
Comment (deepsm007) [inline cmd/ci-images-mirror/main.go:312]: This might be incorrect.

Comment (deepsm007) [reply cmd/ci-images-mirror/main.go:312]: It should be `Info`, a typo I suppose

Comment (hongkailiu) [reply cmd/ci-images-mirror/main.go:312]:
```
Infof("%s...
```
the `f` is there because i used the format string `%s`.

Comment (deepsm007) [reply cmd/ci-images-mirror/main.go:312]: makes sense.

Comment (hongkailiu) [reply cmd/ci-images-mirror/main.go:312]: Maybe using withField and %s together is a bad styling but it is fine syntactically.

**Pull request merged**